03/app.js

```diff
@@ -1,2 +1,13 @@
 const n = 24;
-const oddNumbers = [];
\ No newline at end of file
+const oddNumbers = [];
+// z użyciem pętli for
+function pushToArray(num, arr){
+        for(let i = 1; i <= num; i++){
+                if(i % 2 !== 0){
+                   arr.push(i);     
+                }
+        }
+        return arr
```

Comment:
Warto zwrócić uwagę, że w momencie przekazywania tablicy przez parametr wykonanie `arr.push()` t tak na prawdę `oddNumbers.push()`. To tzw. referencja. Trzeba na to uważać i działa to z typami złożonymi.